### DISCUSSION TITLE
Why not leptos?

### DISCUSSION THREAD

**Discussion by the-gazelle** [lapce/floem#498] [Q&A] [answered]
Hi all,
Total noob here, but as I'm reading about floem, I can't help but wonder "why not use leptos, and combine efforts?" Rather than creating yet another rust ui framework. Is it because leptos is specifically targeted for web?
Just trying to understand more about UI frameworks in rust!
Thanks in advance!

**Comment by jrmoulton** [comment, marked as answer]
>  Is it because leptos is specifically targeted for web?

Yes. Floem is targeting native, leptos targets web.

The two libraries really don't have much in common

**Reply by the-gazelle** [reply]
Gotcha, thanks for clarifying!
